GestionareStructuraEchipaDeFormula1/src/gestionareStructuraEchipaDeFormula1/Servicii.java

```diff
@@ -0,0 +1,108 @@
+package gestionareStructuraEchipaDeFormula1;
+
+import gestionareStructuraEchipaDeFormula1.Angajat.AltJob;
+import gestionareStructuraEchipaDeFormula1.Angajat.Angajat;
+import gestionareStructuraEchipaDeFormula1.Angajat.Pilot;
+import gestionareStructuraEchipaDeFormula1.Angajat.StaffCursa;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Servicii {
+
+    private ArrayList<Angajat> angajati = new ArrayList<>();
+    private ArrayList<Pilot> piloti = new ArrayList<>();
+    private ArrayList<StaffCursa> staff = new ArrayList<>();
+    private ArrayList<AltJob> altejoburi = new ArrayList<>();
+    private ArrayList<Fabrica> fabrici = new ArrayList<>();
+    private ArrayList<Magazin> magazine = new ArrayList<>();
+    private ArrayList<Sponsor> sponsori = new ArrayList<>();
+    private Map<Integer, Integer> salarii = new HashMap<>();
```

Comment:
please use List for references instead of ArrayList:
private List<Angajat> angajati = new ArrayList<>();